### ISSUE TITLE
Part IV: Selection criteria, C: Technical and professional ability, For works contracts: performance of works of the specified type

### ISSUE THREAD

**Issue by AJDAKOST** [OP-TED/ESPD-EDM#288]
EO`s side

Part IV: Selection criteria, C: Technical and professional ability, For works contracts: performance of works of the specified type

What is the difference between Total amount and Specific amount and why both field have to be mandatory?

**Comment by hricolor** [comment]
Hi, 

the difference between both terms is that the total amount includes the whole value of the contract (including shared with other EO). Instead, the specific only includes the value of the contract that the EO carried out. Note that the specific and total amount can be the same (the EO probably executed the work alone, as a sole contractor). 

Regarding the point of both fields being mandatory, it is worth to note that in the current ESPD 2.1.1, both are optional.

For more information see the following link: https://espd.github.io/ESPD-EDM/v2.1.1/xml_guide.html#xml-example-extended-requirements-and-questions, which is an example of this specific criterion.

**Comment by hricolor** [comment]
Dears, 

Just to ensure that this is clear, here it is an example of how it could be interpreted: 

I’m an EO that last year participated in a project for building a hospital. I participated as an architect. 

The whole project has 1.000.000 euros, and I as the architect I got 200.000 euros. 

Therefore, the total amount of the project is 1.000.000 euros, while the specific amount is 200.000 euros. 

Also, it is worth to note that the specific amount cannot be higher than the total. 

Hope this help to have a clearer vision. 

Thank you!!
